src/renderer/lib/migrations.js

```diff
@@ -95,15 +91,15 @@ function migrate_0_7_0 (saved) {
 }
 
 function migrate_0_7_2 (saved) {
-  if (!saved.prefs) {
```

Comment:
Was this a bug before? Seems like both ways should work.

Usually I wouldn't change migration steps that have already run